beta-pipelines/multiome/atac.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ATAC.fastq_gzipped_input_read1": "File",
+  "ATAC.trim_disk_size": "Int? (optional)",
```

Comment:
remove the optional inputs if you are not specifying them.

Reply:
Done!